SIL.Archiving/RampArchivingDlgViewModel.cs

```diff
@@ -1674,41 +1674,34 @@ private void CreateZipFile(CancellationToken cancellationToken)
 
 			ReportMajorProgressPoint(StringId.SavingFilesInPackage, cancellationToken);
 
-			using (var zip = new ZipFile())
+			using (var zip = ZipFile.Open(PackagePath, ZipArchiveMode.Create))
 			{
-				// RAMP packages must not be compressed or RAMP can't read them.
```

Comment:
have we tested the zip files made with this change to ensure RAMP can read them? it sounds like it's picky

Reply:
I just tested it and RAMP seems to be completely happy with the package as zipped.
<!-- Reviewable comment -OBlS8mn6lZgm0bo5cy3:b9rshlt -->
<!-- Sent from Reviewable.io -->